src/rpcblockchain.cpp

```diff
@@ -250,11 +250,12 @@ Value getblockhash(const Array& params, bool fHelp)
 
 Value getblock(const Array& params, bool fHelp)
 {
-    if (fHelp || params.size() < 1 || params.size() > 2)
+    if (fHelp || params.size() < 1 || params.size() > 3)
         throw runtime_error(
-            "getblock \"hash\" ( verbose )\n"
+            "getblock \"hash\" ( verbose ) ( header only )\n"
             "\nIf verbose is false, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
             "If verbose is true, returns an Object with information about block <hash>.\n"
+            "If header only is true (and verbose is false), limits the output to only the block header.\n"
             "\nArguments:\n"
             "1. \"hash\"          (string, required) The block hash\n"
             "2. verbose           (boolean, optional, default=true) true for a json object, false for the hex encoded data\n"
```

Comment:
headeronly (boolean) would need to be mentioned in the arguments here.

BTW: what about adding a `getblockheader` call, instead of adding a boolean argument? I'd slightly prefer that for readability/simplicity. It would only result in minimal code duplication.